phpstan.neon

```diff
@@ -0,0 +1,5 @@
+includes:
+  - vendor/mglaman/phpstan-drupal/extension.neon
+  - vendor/mglaman/phpstan-drupal/rules.neon
+parameters:
+  customRulesetUsed: true
```

Comment:
Same here. Let's merge tool config separately in another PR. They are different things and we don't want to hold up one of them for another.

Reply:
This is also merged. I will rebase with main.

Reply:
Please see point 2 in PR https://github.com/contrib-tracker/backend/pull/443